torch/csrc/jit/python/python_sugared_value.cpp

```diff
@@ -306,6 +306,34 @@ std::shared_ptr<SugaredDict> ModuleValue::getSugaredNamedBufferDict(
       std::make_shared<SugaredTupleValue>(values));
 }
 
+std::shared_ptr<SugaredDict> ModuleValue::getSugaredNamedParametersDict(
```

Comment:
I am pretty sure named_parameters, named_buffers, etc. are recursive. I don't see the code to handle that here.

Reply:
Hey, I opened #38863 to address this in a subsequent diff.

Reply:
Are you fine with me landing it as-such, and then adding the recursive implementation after?